openapi.yaml

```diff
@@ -3637,6 +3642,7 @@ components:
                     description: A bool representing whether or not access requests to the group require an access ticket.
                     example: False
                     type: boolean
+                    x-speakeasy-ignore: true
```

Comment:
nixed because `require_support_ticket` should come from the request_configuration. top level field is deprecated